action.yml

```diff
@@ -75,6 +79,11 @@ runs:
             MSG_GH_PR=""
           fi
 
+          # Sanitize commit message if requested
+          if [[ "${{inputs.sanitize_commit_message}}" == "true" ]]; then
+            MSG_OLD=${MSG_OLD//---/...}
```

Comment:
So this would replace all `---` with `...`?

Can you add the task Nr. to the comment for future reference? What would change now look like on the quety/gui side in gerrit?

Reply:
> So this would replace all `---` with `...`?
Yes, it would. I thought about making `...` a parameter but I think it's not necessary for now. 

> Can you add the task Nr. to the comment for future reference? What would a change now look like on the query/gui side in gerrit?
I have added task's phabricator link to the PR comment, if you mean that. Otherwise I couldn't get it.

Reply:
I meant the comment above, but yeah the PR comment will be committed as well so forget what I said XD

Reply:
Does this mean when a commit message contains e.g. `----`, that will be changed to `...-`? I really can't tell if this is what needs to be done.
* What "problems" are we talking about?
* Why can't we fix the code that gets confused by such dashes? Wouldn't this be a cleaner, less confusing solution?
* If that's not possible, can't we change the code that adds these dashes and make it add something else instead?
* Does the position of the dashes matter? For example, if only dashes at the start of a line cause problems, the regex should only touch dashes that are at the start of a line (probably done via `/^---/`).
* Is the number of dashes relevant?

Reply:
@thiemowmde it means that yes and I'm not very familiar with bash script operations, I'd be happy if you have suggestions. 

* You can find current problem in this ticket: https://phabricator.wikimedia.org/T295601
* It's gerrit's `commit-msg` hook, so it's not likely to change
* And the code adds these dashes is `dependabot` and it's not configurable, so this one is not likely neither
* `commit-msg` uses `git-interpret-trailers` internally, [further investigation in Phabricator](https://phabricator.wikimedia.org/T295601#7732756)